lale/lib/sklearn/ordinal_encoder.py

```diff
@@ -318,7 +318,7 @@ def inverse_transform(self, X):
 When this parameter is set to `ignore` and an unknown category is encountered during transform,
 the resulting encoding with be set to the value indicated by `encode_unknown_with` (this functionality is added by lale).
 """,
-                values=["error", "ignore", "use_encoded_value"],
```

Comment:
This might have to be left as is. The value `ignore` was used to apply our own custom fix for an unknown value.

Reply:
Thank you. I will add it back.